antismash/detection/hmm_detection/data/truD.hmm

```diff
@@ -0,0 +1,496 @@
+HMMER3/f [3.1b2 | February 2015]
+NAME  truD_seed
```

Comment:
This should probably be called `truD` just like in the cluster rules and the hmmdetails file.

Reply:
Changed now!